components/director/internal/domain/bundlereferences/repository.go

```diff
@@ -196,14 +201,15 @@ func (r *repository) ListByBundleIDs(ctx context.Context, objectType model.Bundl
 		return nil, nil, err
 	}
 
-	isInternalVisibilityScopePresent, err := scope.IsGivenScopePresent(ctx, internalVisibilityScope)
+	isInternalVisibilityScopePresent, err := scope.Contains(ctx, internalVisibilityScope)
 	if err != nil {
-		return nil, nil, err
+		isInternalVisibilityScopePresent = true
 	}
 
+	visibilityFilteringSubquery := fmt.Sprintf("(SELECT %s FROM %s WHERE %s.id = %s.%s)", visibilityColumn, objectTable, objectTable, BundleReferenceTable, objectIDCol)
```

Comment:
Why don't use the query builder? I think in most cases we use the query builer to generate SQL for subqueries.
This way the SQL crafting will be isolated there.

Reply:
Because the subquery itself is pretty straightforward - it does not contain any arguments that may need rebinding or some additional subquery (e.g. tenant isolation). That's why, I decided the effort of initialising, constructing and building the subquery via builder is not worth it.